IS0801Test.py

```diff
@@ -12,12 +12,405 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from GenericTest import GenericTest
+from GenericTest import GenericTest, NMOSTestException
+from TestHelper import compare_json
+from TestResult import Test
+from NMOSUtils import NMOSUtils
+from is08 import Action, Activation
+from is08.inputs import getInputList
+from is08.outputs import getOutputList
+from is08.active import Active
+from is08.io import IO
+from is08.testConfig import globalConfig
+import time
+import copy
+
+MAPPING_API_KEY = "channelmapping"
 
 
 class IS0801Test(GenericTest):
     """
     Runs IS-08-01-Test
     """
+
     def __init__(self, apis):
         GenericTest.__init__(self, apis)
+        globalConfig.apiUrl = apis[MAPPING_API_KEY]['url']
+        globalConfig.testSuite = self
+        globalConfig.apiKey = MAPPING_API_KEY
+
+    def test_01(self):

```

Comment:
It would be good if completely new test sets could adopt the naming style from https://github.com/AMWA-TV/nmos-testing/blob/master/IS0502Test.py just so that https://github.com/AMWA-TV/nmos-testing/issues/24 doesn't become a bigger job.